core/templates/pages/exploration-editor-page/settings-tab/settings-tab.component.spec.ts

```diff
@@ -250,6 +251,52 @@ describe('Settings Tab Component', () => {
       flush();
     }));
 
+  it('should able to add or remove exploration editor tags', fakeAsync(() => {
```

Comment:
We seem to be testing multiple scenarios in 1 test. Can you either split them into different tests?

Reply:
done